nservicebus/azure-service-bus/oversized-sends.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Azure Service Bus Oversized Sends
+reviewed: 2016-08-29
+versions: '[7,)'
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Custom handling of oversized Sends
+
+As mentioned, [in error handling](error-handling.md#message-size-problems), the ASB SDK has a peculiar behavior when it comes to computing the message size, resulting in the possibility that oversized messages get sent even when not intended.
```

Comment:
Would calling it "odd" help? :)
The size is not calculated properly until it's too late :disappointed: